app/helpers/static_pages_helper.rb

```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```

Comment:
If you aren't using a particular file, don't include it. This is an empty module. There is no need to push this file.